### PR TITLE
Rename ChooseBest output `likelihoods` to `scores`

### LINKED ISSUE #20: ChooseBest output should be called 'scores'
Part of cohere-ai/blobheart#1351

### PR DESCRIPTION
**Pull request by robertkozin**
Closes #20

### CODE REVIEW

File: cohere/best_choices.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 from typing import List
 
 class BestChoices:
-    def __init__(self, likelihoods: List[float], mode: str) -> None:
-        self.likelihoods = likelihoods
+    def __init__(self, scores: List[float], mode: str) -> None:
+        self.scores = scores
         self.mode = mode
         self.iterator = iter(likelihoods)
```
Comment (jimwu6) [inline cohere/best_choices.py:7]: missed in the iter

Comment (jimwu6) [reply cohere/best_choices.py:7]: and in str representation

Comment (robertkozin) [reply cohere/best_choices.py:7]: Good catch, thanks :)

**Review by jimwu6 — APPROVED** [review]
LGTM

**Comment by robertkozin** [comment]
> LGTM

Haha thank you for fixing the tests, I was rushing to change this and missed too much, that's my bad!

I'm going to merge this once the API level change is merged, sound good?

**Pull request merged**